lib/Arch/Runtime/HyperCall.cpp

```diff
@@ -42,15 +42,13 @@ Memory *__remill_sync_hyper_call(State &state, Memory *mem,
       break;
 
     case SyncHyperCall::kX86LoadGlobalDescriptorTable:
-      asm volatile("lgdt"
-                   :
-                   : "=m"(__remill_read_memory_64(mem, state.addr_to_load)));
+      auto read = __remill_read_memory_64(mem, state.addr_to_load);
```

Comment:
I believe that you'll need parenthesis around the switch case, otherwise you'll get compiler warnings or possibly errors.

Reply:
Do you mean braces around the `case`? It seems to be building fine.